specification/devcenter/DevCenter/DevBox/models.tsp

```diff
@@ -299,15 +394,33 @@ is detected.
   gracePeriodMinutes?: int32;
 }
 
+@doc("The Schedule list result.")
+model ScheduleListResult is Azure.Core.Page<Schedule>;
+
 @doc("A Schedule to execute action.")
 @resource("schedules")
 @parentResource(Pool)
 model Schedule {
+  @doc("The unique URI of the schedule.")
+  @added(APIVersions.v2024_02_01)
+  uri: url;
```

Comment:
Should this be optional?

Reply:
There are several `uri` properties added on various models should they actually be optional? For back compat, they probably should be

Reply:
The one below on DevBox is marked as optional which is what I would expect

Reply:
hmmm, i looked in the previous swagger and they all show as required parameter

Reply:
Got it, then is the DevBox uri incorrect? Should it also be required?

Reply:
So, for all GET operations we will always return the uri, including the dev box. But because DevBox you can also create, they have made only made `poolName` as required property

Reply:
So i guess URI should be required with read only in this case